.buildkite/ftr_configs.yml

```diff
@@ -18,6 +18,8 @@ disabled:
   - x-pack/test/security_solution_api_integration/config/serverless/config.base.essentials.ts
   - x-pack/test/security_solution_endpoint/config.base.ts
   - x-pack/test/security_solution_endpoint_api_int/config.base.ts
+  - x-pack/test/observability_solution_api_integration/config/serverless/config.base.ts
```

Comment:
Is this enough to make the new tests part of CI pipeline?

Reply:
It is. lgtm